src/App.js

```diff
@@ -18,6 +19,7 @@ function App() {
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path="/Sponsors" element={<Sponsors />} />
+          <Route path="/" element={<About />} />
```

Comment:
Hey Queenly! Looks great, I just noticed one problem. Make sure to change "/" to "/About" and we should be good to go. You can just commit to this branch again with the change made, and it should update the PR.

Reply:
Hi Mark, I made the changes, but I'm not sure if it updated into this PR, could you let me know?